CRM/Contact/BAO/GroupContactCache.php

```diff
@@ -67,17 +67,16 @@ public static function check($groupIDs) {
    */
   public static function groupRefreshedClause($groupIDClause = NULL, $includeHiddenGroups = FALSE) {
     $smartGroupCacheTimeout = self::smartGroupCacheTimeout();
-    $now = CRM_Utils_Date::getUTCTime();
+    $expiryTime = date_format(date_create(CRM_Utils_Date::getUTCTime(-($smartGroupCacheTimeout * 60))), 'Y-m-d H:i:s');
```

Comment:
I think we should convert these fields to timestamp - converting to utc time is a poor man's timestamp field (except timestamp fields are actually free) and just leave room to error